### PR TITLE
Fill home page with a general overview

### PR DESCRIPTION
**Pull request by larsblumberg**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1,32 +1,77 @@
...
-    senic-hub/index
+.. image:: hub-software-components.png
+   :scale: 80 %
 
+Technical Details
+=================
 
-senicOS
-=======
+It took us many technical decisions to design the Senic Hub in a way that is not only powerful, but also easy to use, program and update. Some decisions were right clear while others required a thorough understanding of what our smart home users need and what we can build with fairly limited resources of a start up. In the following we are presenting some of the most decisions we took and we are happy to share our learning along the process.
```
Comment (grunskis) [inline index.rst]: "Some decisions were right clear" - did you mean "Some decisions were clear right from the beginning"?

Comment (grunskis) [inline index.rst]: "In the following **\<missing a noun here\>** we are presenting some of the **most** (is it some or most?) decisions we"

File: index.rst [hunk 2]
```diff
@@ -1,32 +1,77 @@
...
+
+Operating System: Linux
+-----------------------
+
+.. image:: linux-tux.png
+   :align: right
+   :width: 100 px
+   :height: 118 px
+
+We are building on top of awesome open-source systems. Linux is a great open-source operating system, it's considered very stable and has support for various hardware platforms. We're using the latest Linux Kernel 4.10 and we are building our own distribution to have full control over which tools become part of the Linux subsystem. Having as few as possible tools running lets us reduce the amount of moving the parts and thus achieve a high system availability.
```
Comment (grunskis) [inline index.rst]: Should we already mention that we're using Yocto here?

File: index.rst (hunk 1 above)
Comment (larsblumberg) [inline index.rst]: Good catch! I changed to _we are presenting the most important decisions_.

_In the following we are presenting_ should be correct English though.

File: index.rst (hunk 2 above)
Comment (larsblumberg) [inline index.rst]: 👍 Maybe @tomster can suggest a sentence?

**Comment by larsblumberg** [comment]
@grunskis I decided to use `music component` instead of `media player component` as the latter name is too long and I consider it a detail that it's called `media player`. Happy to change it later should it turn out to be a problem.

File: index.rst (hunk 1 above)
Comment (larsblumberg) [inline index.rst]: Thanks – added _from the beginning_

**Pull request merged**